src/shims/unix/android/thread.rs

```diff
@@ -16,18 +16,18 @@ pub fn prctl<'tcx>(
     args: &[OpTy<'tcx>],
     dest: &MPlaceTy<'tcx>,
 ) -> InterpResult<'tcx> {
-    // We do not use `check_shim` here because `prctl` is variadic. The argument
-    // count is checked bellow.
-    ecx.check_abi_and_shim_symbol_clash(abi, Conv::C, link_name)?;
+    // `prctl` is variadic. The argument count is checked below.
+    let (fixed_args, varargs) =
+        ecx.check_shim_variadic::<1>(abi, Conv::C, link_name, "prctl", args)?;
```

Comment:
```suggestion
    let ([op], varargs) =
        ecx.check_shim_variadic(abi, Conv::C, link_name, "prctl", args)?;
```
Same everywhere else -- you should never use turbofish here.